webpack/scenes/ContentViews/Details/ContentViewDetailActions.js

```diff
@@ -42,7 +44,7 @@ const cvUpdateSuccess = (response, dispatch) => {
   }));
 };
 
-export const updateContentView = (cvId, params) => async dispatch => dispatch(put({
+export const updateContentView = (cvId, params) => dispatch => dispatch(put({
   type: API_OPERATIONS.PUT,
```

Comment:
Weird thing is the update controller method invokes a (blocking) foreman task. So I can see how this was assumed an async operation. But this is correct. 👍🏽